### ISSUE TITLE
[Feature Request] Pause Playlist instead of stopping when switching between combat and token playlist

### ISSUE THREAD

**Issue by Hufnageln** [elizeuangelo/fvtt-module-combat-music-master#18]
Pause Playlist instead of stopping.

Currently if you have music switch between a few playlists they will stop each time the playlist switches. could there be an option to pause the playlist instead of stop it?

**Comment by koricariftaxe** [comment]
Yes, it will pause, unless they are a playlist. That's the issue I think we are wishing was changed, making it so that it can pause single tracks AND playlists. The inability to pause playlists makes it hard to really use a generic playlist at all, in a satisfactory way.
